sdcm/monitorstack/restore.py

```
@@ -0,0 +1,551 @@
+    if not monitoring_stack_dir:
+        LOGGER.error("%s is empty. No data found", base_dir)
+        return False
+    return os.path.join(base_dir, monitoring_stack_dir[0])
+
+
+def get_monitoring_stack_scylla_version(monitoring_stack_dir):
+    try:
+        with open(os.path.join(monitoring_stack_dir, "monitor_version"), encoding="utf-8") as f:
+            versions = f.read().strip()
+        monitoring_version, scylla_version = versions.split(":")
+        if not requests.head(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)